lib/engine/tile.rb

```diff
@@ -305,7 +309,7 @@ def compute_city_town_edges
 
       # populate ct_edges and edge_count as described in above comments
       paths.each do |path|
-        next unless (ct = path.city || path.town)
+        next unless path.exits.any? && (ct = path.city || path.town)
```

Comment:
i don't think this is necessary. if there are no exits the iterator won't do anything

Reply:
Roger